charts/airflow/templates/_helpers/pods.tpl

```diff
@@ -135,6 +139,16 @@ EXAMPLE USAGE: {{ include "airflow.init_container.install_pip_packages" (dict "R
   volumeMounts:
     - name: home-airflow-local
       mountPath: /opt/home-airflow-local
+
+{{- /* user-defined (global) */ -}}
+{{- if .Values.airflow.extraVolumeMounts }}
+{{ toYaml .Values.airflow.extraVolumeMounts | indent 4 }}
+{{- end }}
+
+{{- /* user-defined */ -}}
+{{- if .extraVolumeMounts }}
+{{ toYaml .extraVolumeMounts | indent 4 }}
+{{- end }}
```

Comment:
Please indent these like other sections (so that the `{{- if ...` is at the same indentation as the volumeMount itself would be)

Reply:
I used **| indent 4**. It seems more clean.
Otherwise, I can indent as you requested.